pages/glossary/interacted-with.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Interacted with
+key: interacted-with
+---
+
+Any HTML element:
+
+- with one of the following [semantic roles][semantic role]: `checkbox`, `combobox`, `listbox`, `menuitemcheckbox`, `menuitemradio`, `radio`, `searchbox`, `slider`, `spinbutton`, `switch` and `textbox`;
+- that, either through user interaction or programmatically, has had its [dirty value flag][] set to true.
```

Comment:
This flag is only available for native input / select / textarea elements. That's not what the roles suggest.

Reply:
This definition is no longer used. I'll remove it.